changelogs/current.yaml

```diff
@@ -352,3 +352,7 @@ deprecated:
   change: |
     deprecated runtime key ``overload.global_downstream_max_connections`` in favor of :ref:`downstream connections monitor
     <envoy_v3_api_msg_extensions.resource_monitors.downstream_connections.v3.DownstreamConnectionsConfig>`.
+- area: http ext_proc filter
+  change: |
+    Change to hornor HeaderValueOption.change_action. This change can be reverted temporarily by
```

Comment:
Please use :ref:`append_action <envoy_v3_api_field_config.core.v3.HeaderValueOption.append_action>` to make it clear.

Reply:
Thanks for the comment, I will address it accordingly.

Reply:
I have updated the code accordingly, can you please review again.